File: src/main/scala/arimitsu/sf/cql/v3/messages/Ready.scala
package arimitsu.sf.cql.v3.messages

import java.nio.ByteBuffer

/**
 * Created by sxend on 14/06/07.
 */
case class Ready() {
}

object ReadyParser extends ResponseParser[Ready] {
  override def parse(body: ByteBuffer): Ready = {
    Ready()
  }
}

File: src/main/scala/arimitsu/sf/cql/v3/messages/Ready.java
package arimitsu.sf.cql.v3.messages;

import arimitsu.sf.cql.v3.Parser;

import java.nio.ByteBuffer;

/**
 * Created by sxend on 14/06/07.
 */
public class Ready {
    private static Ready INSTANCE = new Ready();
    public static final Parser<Ready> ReadyParser = new Parser<Ready>() {
        @Override
        public Ready parse(ByteBuffer body) {
            return INSTANCE;
        }
    };
}
